NSS-Day-Cohort-69/tabloid-one-more-div
# Jh feature/28/view a posts comments

### **Purpose**
To allow a user to view the comments for a post.

### **Files Changed**
- Added CommentController.cs to get comment data
- Added the Subject property to Comment.cs
- Added the Subject property to CommentDTO.cs
- Added the Subject property to CommentForPostDTO.cs
- Added the FormattedDateCreated property to CommentForPostDTO.cs
- Added seed data for the Subject property in the TabloidDbContext.cs
- Added UserName with IdentityUser to UserProfileForCommentDTO.cs
- Added the CommentList.jsx Route to ApplicationViews.jsx
- Added the CommentList.jsx component to view all comments for a post
- Added a link from the PostDetails.jsx Comments button to the CommentList.jsx component using useNavigate
- Added the commentManager.js to access the comments data

### **To Test**

Fetch, setup, and run according to project README, then confirm the following

- Logged in user can navigate to the comments view from the post details
- Logged in user can view all comments for the post ordered by date
- Logged in user can see the edit and delete buttons if they are the author of the comment

Closes #28
